### PR TITLE
Ontology: New widget

### PR DESCRIPTION
**Pull request by VesnaT**
##### Description of changes
- import ontology from file
- edit ontology
- save ontology to file

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ajdapretnar** [comment]
As a replacement of #758?

**Comment by VesnaT** [comment]
> As a replacement of #758?

yes

File: requirements.txt [hunk 1]
```diff
@@ -26,3 +26,4 @@ tweepy >=4.0.0
 ufal.udpipe >=1.2.0.3
 wikipedia
 yake
+sentence_transformers
```
Comment (PrimozGodec) [inline requirements.txt:29]: I know it is not ready yet. I am just adding the note that we do not forget: sentence_transformers should be removed requirements.

Comment (VesnaT) [reply requirements.txt]: done

**Comment by ajdapretnar** [comment]
Ontology fails when entering the keyword manually, then not pressing Enter yet (not confirming the word) and pressing "Add new word" again.

Error:
```
--------------------------- RuntimeError Exception ----------------------------
Traceback (most recent call last):
  File "/Users/ajda/orange/orange3-text/orangecontrib/text/widgets/owontology.py", line 706, in __on_ontology_data_changed
    self.__update_score()
  File "/Users/ajda/orange/orange3-text/orangecontrib/text/widgets/owontology.py", line 769, in __update_score
    score = round(self.__onto_handler.score(tree), 2) \
  File "/Users/ajda/orange/orange3-text/orangecontrib/text/ontology.py", line 342, in score
    self._get_embeddings(tree.labels, wrap_callback(callback, end=0.7)),
  File "/Users/ajda/orange/orange3-text/orangecontrib/text/ontology.py", line 366, in _get_embeddings
    embs = self.embedder(missing, callback=wrap_callback(callback, start=0.6, end=0.9))
  File "/Users/ajda/orange/orange3-text/orangecontrib/text/vectorization/sbert.py", line 66, in __call__
    result_ = self._server_communicator.embedd_data(chunks, callback=callback)
  File "/Users/ajda/orange/orange3/Orange/misc/server_embedder.py", line 126, in embedd_data
    return asyncio.run(
  File "/Users/ajda/.pyenv-x86/versions/3.9.10/lib/python3.9/asyncio/runners.py", line 33, in run
    raise RuntimeError(
RuntimeError: asyncio.run() cannot be called from a running event loop
-------------------------------------------------------------------------------
```

**Pull request merged**